bgpd/bgp_lcommunity.c

```diff
@@ -537,3 +537,133 @@ void lcommunity_del_val(struct lcommunity *lcom, uint8_t *ptr)
 		i++;
 	}
 }
+
+static unsigned int bgp_aggr_lcommunity_hash_key(void *data)
+{
+	struct lcommunity *lcommunity = data;
+
+	return jhash(lcommunity->val, lcommunity->size, 0xdcba4321);
+}
+
+static bool bgp_aggr_lcommunity_hash_cmp(const void *n1, const void *n2)
```

Comment:
Isn't this lcommunity_cmp?

Reply:
I added a new function to safeguard the aggregation hash just in case the existing function goes through changes in the future.
I've made the changes to use the existing function.